bundle-workflow/src/test_workflow/put_file_into_S3.py

```diff
@@ -0,0 +1,361 @@
+                            stored[i] = val
+                            print(stored[i])
+                        elif i == 7 or i == 8 or i == 9 or i == 10:
+                            val = data['testResults']['operationsSummary'][arg1]['latencyMillis'][values[i]]
+                            stored[i] = val
+                            print(stored[i])
+                        else:
+                            val = data['testResults']['operationsSummary'][arg1][values[i]]
+                            stored[i] = val
+                            print(stored[i])
+                    return stored
+
+                def testResults_Operations(arg1):
+                    if arg1 == 'auto-date-histogram':
+                        arg1 = 'auto-date-histogram'
+                    elif arg1 == 'date-histogram-with-tz':
+                        arg1 = 'date-histogram-with-tz'
+                    elif arg1 == 'date-histogram':
+                        arg1 = 'date-histogram'
+                    elif arg1 == 'auto-date-histogram-with-tz':
+                        arg1 = 'auto-date-histogram-with-tz'
+                    else:
+                        arg1 = 'index-append'
+
+                    for i in range(1, 11):
+                        if i == 4 or i == 5 or i == 6:
+                            val = data['testResults']['operations'][arg1]['requestsPerSecond'][values[i]]
+                            stored[i] = val
+                            print(stored[i])
+                        elif i == 7 or i == 8 or i == 9 or i == 10:
+                            val = data['testResults']['operations'][arg1]['latencyMillis'][values[i]]
+                            stored[i] = val
+                            print(stored[i])
+                        else:
+                            val = data['testResults']['operations'][arg1][values[i]]
+                            stored[i] = val
+                            print(stored[i])
+                    return stored
+
+                def testResults_Stats(arg1):
+                    if arg1 == 'cpuStats':
+                        arg1 = 'cpuStats'
+                    else:
+                        arg1 = 'memoryStats'
+
+                    for i in range(7, 11):
+                        val = data['testResults'][arg1]['overall'][values[i]]
+                        stored[i] = val
+                        print(stored[i])
+                    return stored
+
+                def testResults_Garbage():
+                    place_holder = {1: '1', 2: '2'}
+                    old = data['testResults']['garbageCollection']['overall']['oldGCTimeMillis']
+                    new = data['testResults']['garbageCollection']['overall']['youngGCTimeMillis']
+                    place_holder[1] = old
+                    place_holder[2] = new
+                    print(stored[i])
+                    return stored
+
+                text = """
```

Comment:
`Code Smell: Long chunks of imperative code`
Create functions that encapsulate parts of the view e.g. GarbageCollection, Cpu, Operations so that they can be rearranged, it would reshape this section of the code to read more like an outline of the output which would be useful to see.  And as always smaller code snippets are easier to review and write unit tests for